.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+    - "0.10"
```

Comment:
@thompsnm can you add some additional versions to this?
```
 - "0.12"
 - "0.11"
```

We're already running Node 11 and are probably not far from 12. Also, it would be nice to generate the coverage report and send that to https://coveralls.io.